[IPython] [opengl] Make OpenGL kernel print work in IPython & IDLE (#1303)
taichi-dev/taichi: docs/syntax.rst

        before kernel
        after kernel
        inside kernel
        after sync

>     This is due to the fact that GPU memories are only copied on demand, i.e.
>     only when the output data is accessed, or performance will be harmed.

Reviewer: ```suggestion
    This is because GPU memory is only copied when necessary, for performance considerations.
```
